code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/DatadogMetricsOps.scala

```diff
@@ -51,13 +56,22 @@ object DatadogMetricsOps {
       classifier: Option[String]
     ): F[Unit] = {
       val tpe = terminationType match {
-        case TerminationType.Abnormal => "abnormal"
-        case TerminationType.Error    => "error"
-        case TerminationType.Timeout  => "timeout"
+        case TerminationType.Abnormal => typeTag.tag("abnormal")
+        case TerminationType.Error    => typeTag.tag("error")
+        case TerminationType.Timeout  => typeTag.tag("timeout")
       }
-      val tags = Tag.of("type", tpe) :: classifier.toList.flatMap(classifierTags)
+      val tags = tpe :: classifier.toList.flatMap(classifierTags)
       abnormalCount.inc(tags: _*) >> abnormalLatency.record(Duration.ofNanos(elapsed), tags: _*)
     }
+
+    private def statusToBucketTag(status: Status): Tag =
+      status.code match {
```

Comment:
how about `s"${status.code / 100}xx"`?

Reply:
also, how did this pass compiler? we don't have fatal warnings enabled? I would expect this to fail because of non-exhaustiveness.